BUILDING.md

```diff
@@ -468,3 +468,29 @@ export SWIFTSHADER_LD_LIBRARY_PATH=`pwd`
 ```
 
 Next, go to your Filament repo and use the [easy build](#easy-build) script with `-t`.
+
+## SwiftShader for CI
+
+Continuous testing turnaround can be quite slow if you need build SwiftShader from scratch, so we
```

Comment:
Now that one of us knows how to build Docker image, should we do the same for our regular Linux builds? We currently have to download LLVM and CMake every time.

Reply:
Yes we could this.  We could simply use the top part of the Dockerfile that I already wrote.